web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -55,8 +54,6 @@ export class EditFlowComponent implements OnInit, OnDestroy {
   projectDirectory: string;
   flowEnded: string = '';
   stepType: typeof StepType = StepType;
-  runFlowClicked: boolean = false;
-  disableSelect: boolean = false;
```

Comment:
It looks like some functionality that's on develop has been removed with this PR. Please look into this